docs/src/components/Layout/SecondaryNav.tsx

```diff
@@ -101,12 +101,8 @@ export const SecondaryNav = (props) => {
           <NavLink {...props} href="/theming/dark-mode">
             Dark mode
           </NavLink>
-          <NavLink
-            {...props}
-            platforms={['react', 'vue', 'angular']}
-            href="/theming/alternative-styling"
-          >
-            Alternative styling
+          <NavLink {...props} href="/theming/css-variables">
+            CSS Variables
```

Comment:
This doesn't apply to Flutter, shouldn't we add the platforms?
```
             platforms={['react', 'vue', 'angular']}
```

Reply:
good catch! Done.